### PR TITLE
Speaker model update for voxceleb.

### PR DESCRIPTION
**Pull request by barius**
**Thank you for submitting a pull request! Please provide the following information for code review:**

# Pull Request Summary
Fix speaker mode for voxceleb.

# Test Plan
Locally tested.

### CODE REVIEW

File: delta/utils/solver/estimator_solver.py [hunk 1]
```diff
@@ -86,8 +86,12 @@ def _model_fn(features, labels, mode, params):
 
       if mode == utils.INFER:
         softmax = tf.nn.softmax(logits, name='softmax_output')
+        if self.config['solver']['postproc'].get('output_softmax', False):
```
Comment (zh794390558) [inline delta/utils/solver/estimator_solver.py]: will broken other code.

File: egs/voxceleb/spk/v1/conf/tdnn_softmax.yml [hunk 2]
```diff
@@ -53,7 +53,7 @@ data:
       cmvn_type: none # global, local, sliding
       cmvn_path: exp/delta_speaker/cmvn.npy # cmvn file
     classes:
-      num: 6153
+      num: 7323
```
[the remark this answers is not in the document]
Comment (barius) [reply egs/voxceleb/spk/v1/conf/tdnn_softmax.yml]: I used to do experiments on voxceleb 2 alone. Now added voxceleb 1.

File: delta/utils/solver/estimator_solver.py (hunk 1 above)
Comment (barius) [reply delta/utils/solver/estimator_solver.py]: Okay. Tested on voxceleb, it works fine.
I'll revert it however.

Comment (barius) [reply delta/utils/solver/estimator_solver.py]: Commit reverted.

**Pull request merged**